includes/class-cache.php

```
@@ -870,5 +870,23 @@ private function calculate_directory_size( string $directory ): int {
+		 *
+		 * @since 2.1.0
+		 *
+		 * @param string $group The cache group to flush.
+		 * @return bool True if the flush succeeded or the function is unavailable, false on failure.
+		 */
+		public static function flush_group( string $group ): bool {
+			if ( function_exists( 'wp_cache_flush_group' ) ) {
+				return wp_cache_flush_group( $group );
+			}
+			return false;
+		}
```

PHPMD (2.15.0)

[error] The method flush_group is not named in camelCase. (undefined)

(CamelCaseMethodName)